projects/apple/tomviz.bundle.cmake

```diff
@@ -32,7 +27,7 @@ install(CODE "
          USE_SOURCE_PERMISSIONS TYPE DIRECTORY FILES
          \"${install_location}/lib/python2.7/site-packages/\")
 
-    if(${itk_condition_variable})
+    if(${itk_ENABLED})
```

Comment:
That makes more sense, missed that when reading it. Looks good, thanks for the explanation.